src/bounds.jl

```diff
@@ -34,6 +34,28 @@ function bound_remainder(::Type{TaylorModel1}, f::Function, polf::Taylor1, polfI
     end
     return Δ
 end
+function bound_remainder(::Type{TaylorModel1}, f::Function, polf::Taylor1{TaylorN{T}}, polfI::Taylor1, x0, I::Interval) where {T}
```

Comment:
Could you add some comment to make explicit that we are assuming that the domain of the `TaylorN` part is the  normalized symmetric box?

Reply:
ready!